lib/v2/jewishmuseum/exhibitions.js

```diff
@@ -0,0 +1,16 @@
+const buildData = require('@/utils/common-config');
+
+module.exports = async (ctx) => {
+    const link = 'https://thejewishmuseum.org/exhibitions';
+
+    ctx.state.data = await buildData({
+        link,
+        url: link,
+        title: 'Jewish Museums - Exhibitions',
+        item: {
+            item: '#current article.exhibition, #upcoming article.exhibition, #past article.exhibition',
```

Comment:
```suggestion
            item: '#current article.exhibition, #upcoming article, #past article.exhibition',
```
upcoming exhibition isn't shown